buildsys.cmake

```diff
@@ -194,21 +215,16 @@ endmacro()
 
 # enable C++17
 macro(enable_cpp17)
+  set(CMAKE_C_STANDARD 11)
```

Comment:
Do we want to mix C and C++ standards?

Reply:
C11 is the last C standard, it has nothing to do with C++ standard versioning.

Reply:
I stand corrected, _they_ released C17 and even C23.